lib/Conversion/TritonGPUToLLVM/GCNAsmFormat.cpp

```diff
@@ -164,10 +175,11 @@ std::string GCNInstrExecution::dump() const {
 
   std::string modsRepr = strJoin(modReprs, " ");
   if (!modsRepr.empty()) {
-    os << instrRepr << " " << argsRepr << ", " << modsRepr;
+    os << instrRepr << " " << argsRepr << " " << modsRepr;
   } else {
     os << instrRepr << " " << argsRepr;
   }
+  os << ";";
```

Comment:
This is a start of a comment in GCN ISA so perhaps we do not need it here (and below in the test).

Reply:
The same thing is in PTX code.
If you think this is redundant, I will remove it